configure-ironic.sh

```diff
@@ -46,6 +51,36 @@ connection = mysql+pymysql://ironic:${MARIADB_PASSWORD}@localhost/ironic?charset
 http_url = http://${IRONIC_URL_HOST}:${HTTP_PORT}
 fast_track = ${IRONIC_FAST_TRACK}
 
+EOF
+
+if [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
+    crudini --merge /etc/ironic/ironic.conf <<EOF
+[inspector]
+endpoint_override = https://${IRONIC_URL_HOST}:5050
+power_off = ${INSPECTOR_POWER_OFF}
+$([ ! -z "$CACERT_FILE" ] && echo "cafile = $CACERT_FILE")
+$([ ! -z "$INSECURE" ] && echo "insecure = $INSECURE" || echo "insecure = false")
```

Comment:
I'm worried about supporting insecure. What's the use case for that?

Reply:
I just added support for it, as it is a configuration option, but I can remove it from everywhere